File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int post=0;
        int negt=0;

        for(int n : nums)
        {
            if(n>0)
            {
                post++;
            }
            else if(n<0)
            {
                negt++;
            }
        }
        return max(post,negt);
    }
};

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.py
class Solution(object):
    def maximumCount(self, nums):
        post=0
        negt=0
        for n in nums:
            if(n<0):
                negt+=1
            elif(n>0):
                post+=1
        return max(post,negt)
        """
        :type nums: List[int]
        :rtype: int
        """
        